cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_data_adapters.go

```diff
@@ -8,7 +8,6 @@ import (
 	"strings"
 	"time"
 
-	kappctrlinstalled "github.com/vmware-tanzu/carvel-kapp-controller/cli/pkg/kctrl/cmd/package/installed"
```

Comment:
This dependency (the only one occurrence) was blocking the upgrade path for the kapp-controller itself (it is still pointing to v0.27). I've replaced it with the actual value `%s-%s-values"` of the template + a comment.